### DISCUSSION TITLE
[DOC] question about Minimum Recommended Hardware

### DISCUSSION THREAD

**Discussion by hchyue** [longhorn/longhorn#8692] [Q&A]
![image](https://github.com/longhorn/longhorn/assets/69490488/43e26491-de99-4d8c-9fe9-694f38ea9b45)

my question is:  for HDD/Spinning Disk or similar performance block device, what is  max IOPS per volume, 500 or 250?

what is the meaning  of  500/250 ?

**Comment by c3y1huang** [comment]
> 500/250 max IOPS per volume (1 MiB I/O)

- 500 IOPS: the maximum number of read operations per second.
- 250 IOPS: the maximum number of write operations per second.
- 1 MiB I/O: each I/O operation is 1 MiB in size.

**Reply by hchyue** [reply]
Very clear, thx
